public/js/eventsPage.js

```diff
@@ -97,13 +97,44 @@ function displayEventsHandler(eventsAndHiringPartners) {
     }
 };
 
-async function displayEvents(events, hiringPartners) {
+async function displayEvents(events, hiringPartners){
     events.forEach(async (event) => {
-        await eventGenerator(event, hiringPartners)
-        .then(async (event) => {
-            await displayHiringPartnersAttending(event)
+        await eventGenerator(event, hiringPartners).then(event => {
+            displayHiringPartnersAttending(event)
+        })
+        return event
+    })
+}
+
+async function addEventListenersToHpDelete(event){
+    let hpDeleteForms = document.querySelectorAll(`.hiring-partner input[data-event='${event.id}'`)
```

Comment:
why the square bracket [ ? is that a typo?

Reply:
Done